app/components/partials/Navbar.js

```diff
@@ -1,20 +1,35 @@
 import React from 'react';
 import { Link } from 'react-router';
 
+const iconClose = 'https://cdnjs.cloudflare.com/ajax/libs/material-design-icons/3.0.1/navigation/1x_web/ic_close_white_48dp.png';
+const iconMenu = 'https://cdnjs.cloudflare.com/ajax/libs/material-design-icons/3.0.1/navigation/1x_web/ic_menu_white_48dp.png';
+
+function toggleDropdown() {
+  var img = document.getElementById('dropdown-trigger');
+  img.src = (img.src == iconClose ? iconMenu : iconClose);
+  var x = document.getElementsByClassName('dropdown-content');
+  for (var i = 0; i < x.length; i++) {
+    x[i].classList.toggle('show');
+  }
```

Comment:
Keeping this code is currently confusing for anyone who looks at it. And we can always change it later when we have multiple dropdown-contents.